Main.py

```diff
@@ -0,0 +1,94 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()#Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')  #Imports BigSmoke
+img2 = pygame.image.load('Harambe.jpg')   #Imports Harambe
+img3 = pygame.image.load('Doge.jpg')      #Imports Doge
+img4 = pygame.image.load('YeaBoi.jpg')    #Imports Boi
+img5 = pygame.image.load('Dio.jpg')       #Imports Dio
+img6 = pygame.image.load('Jaguars.jpg')   #Imports Jaguars
+myfont = pygame.font.SysFont("comic", 100) #Imports text font
+Height =800 #Sets dimensions of window
+Width =800
+
+imagelist = [img1, img2, img3, img4,img5, img6] #Picks random from loaded images
+imagelistRan = (random.choice(imagelist))
+
+window = pygame.display.set_mode((Width, Height), 0, 24) #Creates window
```

Comment:
The 0 and 24 in this line. What do they mean? Use of literal values in this manner [should be avoided](http://www.techrepublic.com/article/avoid-using-magic-numbers-and-string-literals-in-your-code/) where possible.